.github/workflows/upload_image.yml

```diff
@@ -44,8 +44,14 @@ jobs:
         run: |
           docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-          make -B TARGET_PLATFORM=$ARCH IMAGE_TAG=$IMAGE_TAG-$ARCH IMAGE_PROJECT=chaos-mesh/chaos-mesh DOCKER_REGISTRY=ghcr.io image-$IMAGE
-      
+          if [ "${IMAGE}" = "chaos-dashboard" ]; then
+            UI=1
+          else
+            UI=0
+          fi
```

Comment:
How about always using `UI=1`? I think it should not bring side effects to other images. PTAL @YangKeao

Reply:
I think we still should provide a switch to choose whether to build&bundle UI assets or not so that it will be easier to test the dashboard server separately in the development.